CHANGES

```diff
@@ -20,6 +20,8 @@ Breaking API Changes
   `blog post <https://finagle.github.io/blog/2016/04/20/scala-210-and-java7/>`_
   for details. ``RB_ID=828898``
 
+* scrooge-core: `c.t.scrooge.TReusableMemoryTransport` now uses TUnboundedByteArrayOutputStream
```

Comment:
please move this up to 4.x since 4.8 already shipped since you started this PR.

also, i think this is a "New Feature" instead of a "Breaking API Change", right?